docs/General/Manuals/client-manual.rst

```diff
@@ -1480,6 +1480,171 @@ quick way to move around a large map in this manner.
+later. To get started, populate the work list with a grouping of things you want to build. This can be a
+collection of city improvements, units and wonders. Once you have the list configured the way you like it,
+right-click on the work list canvas and a pop up menu will show. Start by selecting :guilabel:`Save Worklist`.
+A dialog box will pop up allowing you to give the list a name. If you clear out the work list on the
+:guilabel:`Production` tab and then right-click on the work list canvas, you can pick :guilabel:`Insert
+Worklist` or :guilabel:`Change Worklist` from the sub-menu. :guilabel:`Insert Worklist` will add the items in
+the saved work list to the main work list. :guilabel:`Change Worklist` will clear what is in the main work
+list and replace it with the saved work list.
+
+We will discuss the :guilabel:`Governor` tab in a bit. Moving to the center, we can see the city citizen tile
+output segment and the city's full vision radius highlighted over the main map as highlighted in Figure 48.
+This city is size 3, so this means that 4 tiles can be worked by the citizens. The city center is worked for
+free and then 3 other tiles are being worked. In this example, there is a forest/river+silk tile,
+grassland+irrigation, and a grassland/river tile being worked. You can click on the city center and the client
+will automatically pick the best tiles for net food to aid city growth. You can also click on a tile to remove
+the citizen from the tile and then click another tile to have the citizen work another tile. This is commonly
+referred to a city micromanagement. If you remove a citizen from working a tile, take a look at the top center
+segment. The citizens icon bar will show one entertainer specialist. If you want to change the entertainer to
+a scientist or a taxman, you can click on it in the icon bar to change.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-center.png
+  :align: center
+  :alt: Freeciv21 City Dialog City Center
+  :figclass: align-center
+
+  Figure 48: City Dialog - City Center
+
+Moving right, we can see the :guilabel:`General` tab on the right most panel as highlighted in Figure 49. This
+tab shows information simiar to what is shown on the top center segment, along with units and city
+improvements that the city has produced and is supporting. Hovering your mouse over many of the items at the
+top of the :guilabel:`General` tab will show detailed calculation information on how the net value is
+produced.
+
+.. figure:: ../../_static/images/gui-elements/city-dialog-general.png
+  :align: center
+  :alt: Freeciv21 City Dialog General
+  :figclass: align-center
+
+  Figure 49: City Dialog - General
+
+
+We will discuss the :guilabel:`Citizens` tab in a bit. Moving to the bottom center, you will see a list of all
+the units that are present in the city as highlighted in Figure 50. There can be units present in a city that
```

Comment:
"... of all ~~the~~ units that are present in the city *,* as..." (add comma before "as")